migrate/20250303_add_aws_location.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:aws_location_credential) do
+      column :id, :uuid, primary_key: true
+      column :access_key, String, null: false, collate: '"C"'
+      column :secret_key, String, null: false, collate: '"C"'
+      column :region_name, String, null: false
+      foreign_key :project_id, :project, type: :uuid, null: false
+    end
+
+    alter_table(:location) do
+      add_foreign_key :aws_location_credential_id, :aws_location_credential, type: :uuid, null: true
+    end
+
+    run "INSERT INTO provider (name) VALUES ('aws');"
+
+    run <<~SQL
+      INSERT INTO
+        action_type (id, name)
+      VALUES
+        ('ffffffff-ff00-830e-87ff-f02b80d85dc0', 'Region:view'),   -- rezzzzzzzz021gzzz0aw0v1ew0
```

Comment:
I would name the permissions `AwsLocationCredential` instead of `Region`, unless we plan to use the same permission for non-AWS regions in the future.

Reply:
I used `AwsRegion` instead